pydocx/export/__init__.py

```diff
@@ -4,10 +4,11 @@
     unicode_literals,
 )
 
-from pydocx.export.html import PyDocXHTMLExporter
+from pydocx.export.html import PyDocXHTMLExporter, PyDocXHTMLExporterWithImageResize
```

Comment:
I created the PyDocXHTMLExporterWithImageResize so that I could use it in the method PyDocX.to_html(resize=True), this way we can easily add resize feature to an existing project. If it uses PyDocX.to_html we can simply add that resize parameter. If it's using PyDocXHTMLExporter we can simply add the mixin class.